Automattic/jetpack
# CRM: Segment list pagination doesn't work properly

Fixes https://github.com/Automattic/zero-bs-crm/issues/2803

## Proposed changes:
This PR fixes an issue with the segment list pagination. The SQL query missed the offset calculation, so we got only a +1 element offset instead of a `offset = per_page*page`



### Other information:


## Jetpack product discussion


N/A
## Does this pull request change what data or activity we track or use?


N/A
## Testing instructions:
- Create 4-6 segments with the aim of being able to create a pagination with a small list. It could be with the same parameters.
- Now limit the items per page to 2
![image](https://user-images.githubusercontent.com/9832440/219419293-66b9415b-f9d1-423f-9999-352797659014.png)

In trunk, if you go through the pages, you will see one segment offset per page instead of two.
In this branch, the pagination will work well, the offset will be two.
